src/pybind/mgr/devicehealth/module.py

```diff
@@ -10,6 +10,11 @@
 from threading import Event
 from datetime import datetime, timedelta, date, time
 
+try:
+    iteritems = dict.iteritems
+except:
```

Comment:
might want to appease pep8 here. to be specific, could catch `AttributeError` instead.

Reply:
does importing iteritems from six and using that make sense?